TN92 - Update ICS notebook with histogram associated with min/max forces measured over a night

b1quint opened the pull request.

b1quint committed:
TN92 - Update ICS notebook with histogram

b1quint committed:
Update ICS analysis to incorporate a histogram with analysis over a night

ykangastro commented on notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-092_inertia_compensation_single_slew_hp.ipynb:
```diff
@@ -79,6 +79,7 @@
    "outputs": [],
```
<pre class="ql-syntax" spellcheck="false">Line #37.        max_ax.axvline(450, ls=":", c="red", alpha=0.5, label="Operational limit")
</pre><p>A very minor suggestion: maybe with another color for one of that? (like orange or fainter red color for operation limit, etc). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/136/discussion/#comment-1685045338'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

ykangastro replied:
<p>Maybe need info. for BLOCK-146 as BLOCK-146 and -178 were queried below. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/136/discussion/#comment-1685045339'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

ykangastro reviewed (approved):
I think that there were only two minor issues, so I will approve it as  I do not want to make it pending. Except for two minor points, everything else looks good.

b1quint committed:
Update notebook to address comments from PR#136
- Fixed cells that raised errors
- Changed color/width of the limit lines in the histogram

b1quint commented:
My last commit deals with some problematic cells and changes the color for the operational limit to make a clearer distinction between the two limits. I also made the lines thicker for easier visualization.
